java/core/src/main/scala/com/azavea/pdal/Main.scala

```diff
@@ -0,0 +1,54 @@
+package com.azavea.pdal
```

Comment:
`package io.pdal` or something like that would be fine. No need for hobu, which is just confusing I think.